x-pack/test_serverless/api_integration/test_suites/common/saved_objects_management/relationships.ts

```diff
@@ -81,244 +78,213 @@ export default function ({ getService }: FtrProviderContext) {
         );
         await kibanaServer.savedObjects.cleanStandardList();
       });
+      describe('searches', () => {
+        it('should validate search relationships schema', async () => {
+          const { body } = await supertest
+            .get(relationshipsUrl('search', '960372e0-3224-11e8-a572-ffca06da1357'))
+            .set(svlCommonApi.getInternalRequestHeader())
+            .set(roleAuthc.apiKeyHeader)
+            .expect(200);
+          relations1 = body.relations[0];
+          expect(() => {
+            relationSchema.validate(body.relations[0]);
+          }).not.to.throwError();
+        });
 
-      it('should validate search response schema', async () => {
-        const { body } = await supertest
-          .get(relationshipsUrl('search', '960372e0-3224-11e8-a572-ffca06da1357'))
-          .set(svlCommonApi.getInternalRequestHeader())
-          .set(roleAuthc.apiKeyHeader)
-          .expect(200);
-        relations1 = body.relations[0];
-        expect(() => {
-          relationSchema.validate(relations1);
-        }).not.to.throwException();
-      });
-
-      it('should work for searches', async () => {
-        const { body } = await supertest
-          .get(relationshipsUrl('search', '960372e0-3224-11e8-a572-ffca06da1357'))
-          .set(svlCommonApi.getInternalRequestHeader())
-          .set(roleAuthc.apiKeyHeader)
-          .expect(200);
-        relations1 = body.relations[0];
-        relations2 = body.relations[1];
-        expect(relations1).to.eql({
-          id: '8963ca30-3224-11e8-a572-ffca06da1357',
-          meta: {
-            editUrl: '/management/kibana/dataViews/dataView/8963ca30-3224-11e8-a572-ffca06da1357',
-            hiddenType: false,
-            icon: 'indexPatternApp',
-            inAppUrl: {
-              path: '/app/management/kibana/dataViews/dataView/8963ca30-3224-11e8-a572-ffca06da1357',
-              uiCapabilitiesPath: 'management.kibana.indexPatterns',
+        it('should work for searches', async () => {
+          const { body } = await supertest
+            .get(relationshipsUrl('search', '960372e0-3224-11e8-a572-ffca06da1357'))
+            .set(svlCommonApi.getInternalRequestHeader())
+            .set(roleAuthc.apiKeyHeader)
+            .expect(200);
+          relations1 = body.relations[0];
```

Comment:
Tests failed when selecting directly off `body.relations[i]`. Extracted to ensure we're asserting against the same document each time the test runs.